pkg/client/example-runner_test.go

```diff
@@ -17,6 +21,79 @@ func ExampleGetAllCompleteServiceExamples() {
 	// <nil>
 }
 
+func ExampleGetAllCompleteServiceExamples2() {
+
+	allExamples := []CompleteServiceExample{
+		{
+			ServiceExample: broker.ServiceExample{
+				Name:            "Basic Configuration",
+				Description:     "Creates an account with the permission `clouddebugger.agent`.",
+				PlanId:          "10866183-a775-49e8-96e3-4e7a901e4a79",
+				ProvisionParams: map[string]interface{}{},
+				BindParams:      map[string]interface{}{}},
+			ServiceName: "google-stackdriver-debugger",
+			ServiceId:   "83837945-1547-41e0-b661-ea31d76eed11",
+			ExpectedOutput: broker.CreateJsonSchema([]broker.BrokerVariable{
+				{
+					Required:  true,
+					FieldName: "Email",
+					Type:      "string",
+					Details:   "Email address of the service account.",
+				},
+			}),
+		},
+	}
+
+	b, err := json.MarshalIndent(allExamples, "", "\t")
+
+	if err != nil {
+		panic(err)
+	}
+
+	os.Stdout.Write(b)
+	// Output:
+	//	[
+	//	{
+	//		"name": "Basic Configuration",
+	//		"description": "Creates an account with the permission `clouddebugger.agent`.",
+	//		"plan_id": "10866183-a775-49e8-96e3-4e7a901e4a79",
+	//		"provision_params": {},
+	//		"bind_params": {},
+	//		"ServiceName": "google-stackdriver-debugger",
+	//		"ServiceId": "83837945-1547-41e0-b661-ea31d76eed11",
+	//		"ExpectedOutput": {
+	//			"$schema": "http://json-schema.org/draft-04/schema#",
+	//			"properties": {
+	//				"Email": {
+	//					"description": "Email address of the service account.",
+	//					"title": "Email",
+	//					"type": "string"
+	//				}
+	//			},
+	//			"required": [
+	//				"Email"
+	//			],
+	//			"type": "object"
+	//		}
+	//	}
+	//]
+}
+
+func ExampleRunExamplesFromFile() {
```

Comment:
Will this work if you aren't running the server?

Reply:
Test removed.